Core/Core/View/Base/Webview/WebView.swift

```diff
@@ -189,7 +199,14 @@ public struct WebView: UIViewRepresentable {
         
         @objc private func reload() {
             parent.isLoading = true
-            webview?.reload()
+            if webview?.url?.absoluteString.isEmpty ?? true {
+                if let url = URL(string: parent.viewModel.url) {
```

Comment:
What will happen if this condition fails? will it reload the webview?

Reply:
Thank you for bringing this in attention. It’s fixed.